BrowserKit/Sources/SummarizeKit/RectangularFadeMaskLayer.swift

```diff
@@ -0,0 +1,89 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+extension CGPoint {
```

Comment:
If its only used in this file?
```suggestion
fileprivate extension CGPoint {
```

Reply:
right, yes this is used across other files too